lib/Retcon/Store/PostgreSQL.hs

```diff
@@ -148,31 +156,38 @@ instance Store PGStore where
       diffQ = "INSERT INTO retcon_diff (entity, id, is_conflict, content) VALUES (?, ?, ?, ?) RETURNING diff_id"
       opsQ = "INSERT INTO retcon_diff_conflicts (diff_id, content) VALUES (?, ?)"
 
-  resolveDiffs (PGStore conn _) did =
-      void $ execute conn sql (Only did)
+  resolveDiffs (PGStore conn _) did = do
+      execute conn "DELETE FROM retcon_diff_conflicts WHERE diff_id = ?" (Only did)
+      execute conn "UPDATE retcon_diff SET is_conflict = FALSE WHERE diff_id = ?" (Only did)
+      return ()
+
+  reduceDiff (PGStore conn _) patchID resolved = do
```

Comment:
`resolved` doesn't seem to be used. Isn't this the same as `resolveDiff`: it SELECTs all the operations for the patchID, and then deletes them all.